tests/licensedcode/data/plugin_license/sqlite/sqlite.expected.json

```diff
@@ -1,4 +1,60 @@
 {
+  "license_detections": [
+    {
+      "identifier": "blessing-d56f762b-a283-5361-23c4-d3935b7e9c75",
+      "license_expression": "blessing",
+      "count": 136,
+      "detection_log": [
+        "not-combined"
+      ],
+      "matches": [
+        {
+          "score": 100.0,
+          "start_line": 30,
+          "end_line": 35,
+          "matched_length": 42,
+          "match_coverage": 100.0,
+          "matcher": "2-aho",
+          "license_expression": "blessing",
+          "rule_identifier": "blessing.LICENSE",
+          "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/blessing.LICENSE"
+        }
+      ]
+    }
+  ],
+  "license_references": [
+    {
+      "key": "blessing",
+      "short_name": "SQLite Blessing",
+      "name": "SQLite Blessing",
+      "category": "Public Domain",
+      "owner": "SQLite",
+      "homepage_url": "https://sqlite.org/src/artifact/df5091916dbb40e6",
+      "is_builtin": true,
+      "spdx_license_key": "blessing",
```

Comment:
I wonder if we should not force including all the attributes even if empty? That's been the standard for how we report things in SCTK so far

Reply:
This can wait for later

Reply:
@pombredanne do you mean the `is_builtin` flag? That is the only attribute we are adding whenever there are extra licenses installed/added to the index, our previous discussion was we add other attributes too based on CLI options/other optional elements. But apart from this all other attributes are always added to the reference. Note that we have more attributes added on tge release prep branch.

Reply:
I meant all the attributes that are empty. For instance in https://github.com/nexB/scancode-toolkit/blob/f1cc917e002d4269869ffefa47acb1bdfb3d2d56/src/licensedcode/models.py#L393 we do not report these

Reply:
I entered https://github.com/nexB/scancode-toolkit/issues/3182